src/redux/actions/messages.js

```diff
@@ -1,3 +1,4 @@
-import { ADD_MESSAGE } from '../types/messages';
+import { ADD_MESSAGES_MESSAGE, DELETE_MESSAGES_MESSAGE } from '../types/messages';
 
-export const addMessage = (text) => ({type: ADD_MESSAGE, payload: text});
+export const addMessage = (text) => ({type: ADD_MESSAGES_MESSAGE, payload: text});
+export const deleteMessage = (id) => ({type: DELETE_MESSAGES_MESSAGE, payload: id});
```

Comment:
Can we name it just ADD_MESSAGE, DELETE_MESSAGE ? =)

Reply:
MESSAGES is the name of the reducer.
ADD_MESSAGE, DELETE_MESSAGE are too common names and can be used for other reducers.